modular_skyrat/modules/modular_weapons/code/modules/projectiles/guns/energy/energy.dm

```diff
@@ -29,10 +29,10 @@
 // Similar to the HoS's laser. Fires a bouncing non-lethal, lethal and knockdown projectile.
 
 /obj/item/gun/energy/e_gun/cfa_phalanx
-	name = "\improper Mk.II Phalanx Plasma Carbine"
+	name = "\improper Mk.II Phalanx Plasma Blaster"
 	desc = "Fires a disabling and lethal bouncing projectile, as well as a special muscle-seizing projectile that knocks targets down. It has <b><span style='color:purple'>Cantalan Federal Arms</span></b> etched into the grip."
 	icon = 'modular_skyrat/modules/modular_weapons/icons/obj/guns/projectile.dmi'
-	icon_state = "phalanx"
+	icon_state = "phalanx1"
```

Comment:
Make sure you mirror this to the phalanx_empty overlay in the icon, is currently runtiming

Reply:
I did not press post but

it's currently runtiming because the code is trying to apply phalanx1_empty which isn't an overlay that exists yet

Reply:
e